### PR TITLE
BZ1977217 Corrected verbiage in Throttling logs in Fluentd

### PR DESCRIPTION
**Pull request by Wiharris**
For only version 3.11

Bug: https://bugzilla.redhat.com/show_bug.cgi?id=1977217

Direct link to doc preview: https://deploy-preview-41732--osdocs.netlify.app/openshift-enterprise/latest/install_config/aggregate_logging.html#aggregated-fluentd

QA: @anpingli

### CODE REVIEW

**Review by GroceryBoyJr — COMMENTED** [review]
Suggested changes:
First Sentence: 
To make changes to Fluentd, change the configuration and restart the Fluentd pods to apply the changes.
Second Sentence:
To make changes to Elasticsearch, you must first scale down Fluentd and then
scale down Elasticsearch to zero. After making your changes, scale Elasticsearch first and
then scale Fluentd back to its original.”

**Comment by bobfuru** [comment]
Please be sure to squash to a single commit to ready this for merge (in addition to the QE review you already requested.) Otherwise, LGTM!! 🎉

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -1942,10 +1942,10 @@ test-project:
...
 
 
-When you make changes to any part of the EFK stack, specifically Elasticsearch
-or Fluentd, you should first scale Elasticsearch down to zero and scale Fluentd
-so it does not match any other nodes. Then, make the changes and scale
-Elasticsearch and Fluentd back.
+To make changes to Fluentd, change the configuration and restart the Fluentd pods to apply the changes.
+To make changes to Elasticsearch, you must first scale down Fluentd and then
+scale down Elasticsearch to zero. After making your changes, scale Elasticsearch first and
+then scale Fluentd back to its original setting.
```
Comment (bobfuru) [inline install_config/aggregate_logging.adoc:1948]: A note that we generally apply softwraps as we encounter them in existing docs. (A cmd+j should do the trick if you're using Atom.) Not necessary to apply here but just a heads up.

**Comment by bobfuru** [comment]
LGTM!

**Pull request merged**